storage/internal/benchmarks/w1r3.go

```diff
@@ -180,15 +171,7 @@ func (r *w1r3) run(ctx context.Context) error {
 			runtime.GC()
 		}
 
-		client, close, err := getClient(ctx, opts, *r.readResults[i])
-		if err != nil {
-			return fmt.Errorf("getClient: %w", err)
-		}
-		defer func() {
-			if err := close(); err != nil {
-				log.Printf("close client: %v", err)
-			}
-		}()
+		client := getClient(ctx, *r.readResults[i])
```

Comment:
IMO for w1r3 we should probably just use the same client for all steps rather than checking out another. what do you think?